packages/mineral-ui-tokens/scripts/formats.js

```diff
@@ -1,76 +1,101 @@
 const keys = require('./keys');
 
-const keyValueOutput = (k, v) => `  ${k}: ${v},`;
-
 const REGEX_ALIAS_VALUE = /{!(.*)}/g;
 
-const defaultExport = (output) => ['export default {', output, '};'].join('\n');
+const categories = (input) => [
```

Comment:
Curious (literally; I would like to understand) why you're spreading this Set inside an array.

Reply:
It needs to be an array because Set doesn't have a reduce method; if you don't spread the set into the array, the output becomes `"[object Set]": {}`; I'm not fully sure why but I would guess it's because a Set objects is a collection of values- maybe @coldpour can explain further?

Reply:
But the value of `[ ...new Set(input.get('props').map(...) ]` over `input.get('props').map(...)` is...?

Reply:
Experimented a little & if you don't spread the Set into the array, you're just wrapping the Set object in an array instead of spreading the Set values onto the array, e.g.:

```js
[new Set([1, 1, 2, 3])] => [Set(3)]
[...new Set([1, 1, 2, 3])] => [1, 2, 3]
```